cypress/e2e/signIn.cy.js

```diff
@@ -1,25 +1,30 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import SignUpPageObject from '../support/pages/signUp.pageObject';
 import SignInPageObject from '../support/pages/signIn.pageObject';
 import HomePageObject from '../support/pages/home.pageObject';
 
+const signUpPage = new SignUpPageObject();
 const signInPage = new SignInPageObject();
 const homePage = new HomePageObject();
 
 describe('Sign In page', () => {
   let user;
 
-  before(() => {
+  beforeEach(() => {
     cy.task('db:clear');
+  });
+
+  before(() => {
     cy.task('generateUser').then((generateUser) => {
       user = generateUser;
     });
```

Comment:
Ensure that the `generateUser` task is correctly providing user data before the tests are executed. If the user data is not available, the tests might fail.